packages/nc-gui-v2/components/dlg/QuickImport.vue

```diff
@@ -0,0 +1,326 @@
+<script setup lang="ts">
+import { useToast } from 'vue-toastification'
+import { Form } from 'ant-design-vue'
+import type { TableType } from 'nocodb-sdk'
+import type { UploadChangeParam } from 'ant-design-vue'
+import { useI18n } from 'vue-i18n'
+import MdiFileIcon from '~icons/mdi/file-plus-outline'
+import MdiFileUploadOutlineIcon from '~icons/mdi/file-upload-outline'
+import MdiLinkVariantIcon from '~icons/mdi/link-variant'
+import MdiCodeJSONIcon from '~icons/mdi/code-json'
+import { fieldRequiredValidator, importCsvUrlValidator, importExcelUrlValidator, importUrlValidator } from '~/utils/validation'
+import { extractSdkResponseErrorMsg } from '~/utils/errorUtils'
+import { ExcelTemplateAdapter, ExcelUrlTemplateAdapter, JSONTemplateAdapter, JSONUrlTemplateAdapter } from '~/utils/parsers'
+import { useProject } from '#imports'
+const { modelValue, importType } = defineProps<Props>()
+
+const emit = defineEmits(['update:modelValue'])
+
+const { t } = useI18n()
+
+interface Props {
+  modelValue: boolean
+  importType: 'csv' | 'json' | 'excel'
+}
+
+const { tables } = useProject()
+const toast = useToast()
+const activeKey = ref('uploadTab')
+const jsonEditorRef = ref()
+const templateEditorRef = ref()
+const loading = ref(false)
+const templateData = ref()
+const importData = ref()
+const templateEditorModal = ref(false)
+const useForm = Form.useForm
```

Comment:
some spaces would help to make this more readable, otherwise it's a big block of variables that are hard to deciper 😄